Project/frontend/src/App.js

```diff
@@ -1,9 +1,57 @@
 import { Chessboard } from "react-chessboard";
+import { useState } from "react";
 
 export default function App() {
+  const [position, setPosition] = useState('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
+  function clearBoard() {
+    setPosition({});
+  }
+  function startPosition() {
+    setPosition('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
+  }
+  function updatePosition(updatedposition) {
+    setPosition(updatedposition);
+    console.log(updatedposition);
+  }
+  
+  let bturn = {"turn":"black",position};
+  let wturn = {"turn":"white",position};
```

Comment:
Move these into the functions - you want to make sure you have the most up to date position when you submit